server/webhook_parser_misc_test.go

```diff
@@ -136,3 +136,16 @@ func TestWebhookQuotedComment(t *testing.T) {
 		assert.True(t, strings.HasPrefix(w.text, ">"))
 	}
 }
+
+func TestEpicLinkUpdateFormat(t *testing.T) {
+	f, err := os.Open("testdata/webhook-issue-updated-epic-link.json")
+	require.NoError(t, err)
+	defer f.Close()
+	bb, err := ioutil.ReadAll(f)
+	require.Nil(t, err)
+	wh, err := ParseWebhook(bb)
+	require.NoError(t, err)
+	w := wh.(*webhook)
+	require.NotNil(t, w)
+	require.Equal(t, w.headline, "Test User **updated** Epic Field from \"TES-11\" to \"TES-12\" on story [TES-41: Unit test summary 1](https://some-instance-test.atlassian.net/browse/TES-41)")
```

Comment:
This test implies that there is no change done here. Shouldn't the test say this?

```suggestion
	require.Equal(t, w.headline, "Test User **updated** Epic Link from [TES-11](https://some-instance-test.atlassian.net/browse/TEST-11) to [TES-12](https://some-instance-test.atlassian.net/browse/TEST-12) on story [TES-41: Unit test summary 1](https://some-instance-test.atlassian.net/browse/TES-41)")
```

Reply:
I know the test above is formatted that way, but can we clean up the formatting a bit here?

```go
func TestEpicLinkUpdateFormat(t *testing.T) {
	f, err := os.Open("testdata/webhook-issue-updated-epic-link.json")
	require.NoError(t, err)
	defer f.Close()
	
	bb, err := ioutil.ReadAll(f)
	require.Nil(t, err)
	wh, err := ParseWebhook(bb)
	require.NoError(t, err)
	w := wh.(*webhook)
	require.NotNil(t, w)

        require.Equal(t, w.headline, "Test User **updated** Epic Link from [TES-11](https://some-instance-test.atlassian.net/browse/TEST-11) to [TES-12](https://some-instance-test.atlassian.net/browse/TEST-12) on story [TES-41: Unit test summary 1](https://some-instance-test.atlassian.net/browse/TES-41)")
```